src/sonic-config-engine/minigraph.py

```diff
@@ -714,8 +714,10 @@ def parse_xml(filename, platform=None, port_config_file=None):
     results['DHCP_SERVER'] = dict((item, {}) for item in dhcp_servers)
     results['NTP_SERVER'] = dict((item, {}) for item in ntp_servers)
     results['TACPLUS_SERVER'] = dict((item, {'priority': '1', 'tcp_port': '49'}) for item in tacacs_servers)
-
     results['ACL_TABLE'] = acls
+    results['FEATURES'] = {'telemetry': {
+        'status': 'on'
+        }}
```

Comment:
If you want to make the definition multi-line (which will probably be best for readability if we add other services), I suggest the following format:

```
results['FEATURES'] = {
    'telemetry': {
        'status': 'on'
    }
}
```

Reply:
makes sense. Updated